frontend/src/App.js

```diff
@@ -61,177 +62,198 @@ const NotFound = () => (
 )
 
 const App = (props) => {
+  const {
+    updateIsLoading,
+    loginUser,
+    fetchRegistrationManagement,
+    setError,
+    initializeMyGroup,
+    logoutUser,
+    clearRegistrations,
+    isLoading,
+    user,
+  } = props
+
   useEffect(() => {
-    props.updateIsLoading(true)
-    if (!window.location.href.includes('customer-review/')){
-      props.loginUser()
-    }
-    fetchRegistrationManagement()
-    props.updateIsLoading(false)
-    setInterval(() => {
-      console.log('timeout')
-      if (!window.location.href.includes('customer-review/')){
-        loginService.login()
-      }
+    const handleLogin = () =>
+      !window.location.href.includes('customer-review/') && loginUser()
+
+    // TODO: these should prolly use async/await?
+    updateIsLoading(true)
+    handleLogin()
+    fetchRegistrationManagementData()
+    initializeMyGroup()
+    updateIsLoading(false)
+
+    // TODO: what's the purpose of this?
```

Comment:
Figure this out

Reply:
This is just to recheck users login status at every 60 secs (for expired sessions etc). I don't think this is any problem, was so in original version and has been working. Removed comment.

Reply:
I was thinking I could create an issue for this as I think it's a bit clunky design. It results in page updating every 60 seconds and for example the input fields in the time logs page clear out every 60 secs because of this.